backend/pcbuilder/serializers.py

```
@@ -1,56 +1,17 @@
 from rest_framework import serializers
 from django.contrib.auth import get_user_model
-from .models import *
+from .models import User
 
 User = get_user_model()
```

Ruff (0.12.2)

Redefinition of unused `User` from line 3

(F811)